os/include/tinyara/wqueue.h

```diff
@@ -306,8 +306,8 @@ struct work_s {
 	struct dq_entry_s dq;		/* Implements a doubly linked list */
 	worker_t worker;			/* Work callback */
 	FAR void *arg;				/* Callback argument */
-	systime_t qtime;			/* Time work queued */
-	systime_t delay;			/* Delay until work performed */
+	clock_t qtime;			/* Time work queued */
```

Comment:
kernel work queue (named kwork_queue) also uses this structure and it uses clock_systimer instead of clock. But it is right becuase it is in kernel space.
So, let's revert this file changes and do typecasting on user work queue.

Reply:
My previous comment also recommented typecasting.
```
sunghan-chang 4 days ago  Member
The qtime element of work_s has systime_t type at definition. This might be (systime_t)clock();
```